packages/client/src/components/app/forms/InnerForm.svelte

```diff
@@ -408,37 +414,23 @@
     }
   }
 
-  const handleScrollToField = ({ field, block }) => {
+  const handleScrollToField = ({ field }) => {
     if (!field.fieldState) {
       field = get(getField(field))
     }
     const fieldId = field.fieldState.fieldId
     const fieldElement = document.getElementById(fieldId)
     fieldElement.focus({ preventScroll: true })
-    if (block === "start") {
-      const label = document.querySelector(`label[for="${fieldId}"]`)
-      label.scrollIntoView({ behavior: "smooth" })
-    } else {
-      fieldElement.scrollIntoView({
-        behavior: "smooth",
-        block,
-      })
-    }
+    const label = document.querySelector(`label[for="${fieldId}"]`)
+    label.style.scrollMargin = "100px"
+    label.scrollIntoView({ behavior: "smooth", block: "nearest" })
```

Comment:
Could we use the field itself rather than the label? There's a chance that there won't be a label, and that will be more likely with the recent (and more upcoming) form block changes.

Reply:
I tested the scenario for when the label is left blank. The label element still exists and the scroll is successful.

I had to use the label because for some reason it wouldn't scroll to a date picker.